documentation/markdown_files/the-markdown-pages.markdown

```diff
@@ -263,3 +263,22 @@ The two approaches listed above can be mixed, this will soon<sup>(tm)</sup> be d
 [CommonMark]: http://commonmark.org/
 
 [python extension]: https://github.com/hotdoc/hotdoc_python_extension
+
+### Supported YAML metadata
+
+The following table lists all availiable options that can be specified in the
+metadata YAML header.
+
+| Key | Type | Description |
```

Comment:
Please only document the metas that you can describe, I'll do a second round when I have time